recipes/cppfront/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+
+import os
+
+required_conan_version = ">=1.47.0"
+
+class CppfrontConan(ConanFile):
+    name = "cppfront"
+    description = "Cppfront is a experimental compiler from a potential C++ 'syntax 2' (Cpp2) to today's 'syntax 1' (Cpp1)"
+    topics = ("cppfront", "cpp2")
```

Comment:
No need for the name

```suggestion
    topics = ("cpp2")
```